### PR TITLE
fix: change label domain name

### PR DESCRIPTION
**Pull request by b4nst**

### CODE REVIEW

File: pkg/manifest/instance.go [hunk 1]
```diff
@@ -17,8 +17,8 @@ package manifest
 
 const (
 	DeletionPolicyAbandon    = "abandon"
-	InstanceLabel            = "cuebe.loft-orbital.com/instance"
-	DeletionPolicyAnnotation = "cuebe.loft-orbital.com/deletion-policy"
+	InstanceLabel            = "instance.cuebe.loftorbital.com/name"
```
Comment (astraios) [inline pkg/manifest/instance.go:20]: `cuebe` being an open source project, it feels a bit weird to force people external to loftorbital.com to use this label. Are `cuebe-instance-name` and `cuebe-deletion-policy` better alternatives? Or, should we register a domain for `cuebe` and use that for those specific labels (and other stuff 🤷 )

Comment (b4nst) [reply pkg/manifest/instance.go:20]: Right now we don't have a specific organization for open source projects, and put everything under Loft Orbital. This is something that we will be able to change in the future. Providing an upgrade command that just update the labels will not be too hard if we have to come to this.

Comment (astraios) [reply pkg/manifest/instance.go:20]: Feels a bit off to force anyone willing to use the tool to set a "loftorbital.com" label prefix. But we're still in alpha, I think we can survive for now.

Comment (b4nst) [reply pkg/manifest/instance.go:20]: It does not bother me tbh. That's the same when you use a tool from google (with google domain labels) or helm (and the `*helm.sh` labels). Cuebe is maintained by and under Loft Orbital org for now, it seems decent to use Loft Orbital domain.

**Pull request merged**